p2p/src/main/kotlin/maru/p2p/messages/BeaconBlocksByRangeResponseMessageSerDe.kt

```diff
@@ -8,44 +8,40 @@
  */
 package maru.p2p.messages
 
+import maru.compression.MaruCompressor
 import maru.p2p.Message
 import maru.p2p.RpcMessageType
 import maru.p2p.Version
-import maru.serialization.SerDe
+import maru.serialization.compression.MaruSnappyFramedCompressor
+import maru.serialization.rlp.MaruCompressorRLPSerDe
 import maru.serialization.rlp.SealedBeaconBlockSerDe
-import org.apache.tuweni.bytes.Bytes
-import org.hyperledger.besu.ethereum.rlp.RLP
 import org.hyperledger.besu.ethereum.rlp.RLPInput
 import org.hyperledger.besu.ethereum.rlp.RLPOutput
 
 class BeaconBlocksByRangeResponseMessageSerDe(
```

Comment:
This class doesn't have to implement the `MaruCompressorRLPSerDe`. You can just instantiate `MaruCompressorRLPSerDe` and pass it as an `RLPSerDe<BeaconBlocksByRangeResponse>`